lxc/init.go

```diff
@@ -313,6 +322,7 @@ func (c *cmdInit) create(conf *config.Config, args []string) (lxd.InstanceServer
 		opInfo = op.Get()
 	}
 
+	// tomp TODO should this field be renamed?
 	containers, ok := opInfo.Resources["containers"]
```

Comment:
@stgraber this field is returned by the LXD server and seems to contain info about what was created, are you happy for this to stay as "containers", or should I add a field called "virtual-machines" or "instances"?

Reply:
Oh yeah, it's annoying how we have a few of those small API bits sprinkled around that we can't easily change without breaking things :)

This one as far as I can tell, is only used in `copy.go` and `init.go`, so I think we should have it look for both instances and containers for backward compatibility and switch the server to returning instances in the Resources map.